functions.php

```diff
@@ -235,6 +235,44 @@ function custom_class( $classes, $field ) {
 	return $classes;
 }
 
+/**
+ *
+ * Update confirmation message for Gravity Forms.
+ */
+add_filter( 'gform_confirmation', 'custom_confirmation', 10, 3 );
+
+/**
+ *
+ * Return custom confirmation message for Gravity Forms.
+ *
+ * @param string $confirmation The default confirmation message.
+ * @param array  $form The form properties.
+ * @param array  $entry The form data.
+ *
+ * @return string The custom confirmation message.
+ */
+function custom_confirmation( $confirmation, $form, $entry ) {
+	$first_name_field = array_filter(
+		$form['fields'],
+		function( $field ) {
+			return 'name' === $field->type;
+		}
+	)[0];
+
+	$first_name = rgar( $entry, '' . $first_name_field->id . '.3' );
```

Comment:
The format `fieldId + '.3'` is how you get the "First Name" portion of a GF name field, see the [docs](https://docs.gravityforms.com/entry-object/) for more details 🙂